src/agents/models.py

```diff
@@ -21,9 +21,13 @@
     models["claude-3-haiku"] = ChatAnthropic(
         model="claude-3-haiku-20240307", temperature=0.5, streaming=True
     )
+if os.getenv("USE_AWS_BEDROCK") == "true":
+    models["bedrock-haiku"] = ChatBedrock(
+        model_id="anthropic.claude-3-5-haiku-20241022-v1:0", temperature=0.5
+    )
 
 if not models:
-    print("No LLM available. Please set API keys to enable at least one LLM.")
+    print("No LLM available. Please set API keys to enable at least one LLM or enable Bedrock with USE_AWS_BEDROCK=true.")
```

Comment:
Could we change this to `"No LLM available. Please set environment variables to enable at least one LLM"` so we don't need to mention Bedrock specifically in the error?

Reply:
looks better, good call.